e2e/tests/ui/features/@advisory-explorer/advisory-explorer.feature

```diff
@@ -2,6 +2,62 @@ Feature: Advisory Explorer
     Background: Authentication
         Given User is authenticated
 
+# Search for advisories
+    Scenario: Search for an advisory using the general search bar
+        
+        When User searches for an advisory named "<advisoryID>" in the general search bar
+        
+        Then The advisory "<advisoryID>" shows in the results
+
+        Examples:
+            | advisoryID      |
+            | CVE-2024-26308  |
+    
+    Scenario: Search for an advisory using the dedicated search bar
+
+        When User searches for "<advisoryID>" in the dedicated search bar
+        
+        Then The advisory "<advisoryID>" shows in the results
+
+        Examples:
+            | advisoryID      |
+            | CVE-2024-26308  |
+
+    # Advisory Explorer
+    Scenario: Display an overview of an advisory
+
+        When User visits Advisory details Page of "<advisoryID>"
+
+        Then The page title is "<advisoryID>"
+        Then The "Download" action is available
```

Comment:
Maybe the 'action is available' is just redundant?
Given there is just below test which does need this action to be available since it is verifying its effect?

It could be usefull only if we have actions which we do not execute - when their functionality is not covered with explicit test (yet or for whatever other reason).

Reply:
@queria I agree that it's kind of useless here now, but I'd leave it in case we have more actions in the future.